graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -284,10 +290,12 @@ public LookupTableApi createTable(@ApiParam LookupTableApi lookupTable) {
     }
 
     @PUT
-    @Path("tables")
+    @Path("tables/{idOrName}")
```

Comment:
Then I'd say we should at least check the ids for equality to prevent mistakes when passing different ids.